src/main.js

```diff
@@ -1,10 +1,13 @@
-var savedIdeas = [];
+var loggedIdea = [];
```

Comment:
Thanks for updating this to make more sense! I kept thinking savedIdeas was not the right name for this as we started progressing through the project.